package-lock.json

```diff
@@ -14142,8 +14084,7 @@
       "version": "1.0.1",
       "resolved": "https://registry.npmjs.org/mimic-response/-/mimic-response-1.0.1.tgz",
       "integrity": "sha512-j5EctnkH7amfV/q5Hgmoal1g2QHFJRraOtmx0JpIqkxhBhI/lJSl1nMpQ45hVarwNETOoWEimndZ4QK0RHxuxQ==",
-      "dev": true,
-      "optional": true
+      "dev": true
```

Comment:
i think these other changes are unnecessary so maybe they should be reverted?

Reply:
These are auto-generated.  I suggest to leave it as is.

Reply:
It's auto-generated and is tedious to manually edit those unnecessary change.  I suggests to remain as is.

Reply:
Umm, I think if you just reset to what's in master and just update the redux SHA, that should suffice.